src/scale/Log.ts

```diff
@@ -97,15 +109,17 @@ class LogScale extends Scale {
     getExtent() {
         const base = this.base;
         const extent = scaleProto.getExtent.call(this);
-        extent[0] = mathPow(base, extent[0]);
-        extent[1] = mathPow(base, extent[1]);
 
         // Fix #4158
         const originalScale = this._originalScale;
         const originalExtent = originalScale.getExtent();
-        this._fixMin && (extent[0] = fixRoundingError(extent[0], originalExtent[0]));
-        this._fixMax && (extent[1] = fixRoundingError(extent[1], originalExtent[1]));
-
+        // To get extent without precision problem, get the original extent directly
+        extent[0] = (extent[0] === mathLog(originalExtent[0]) / mathLog(base))
```

Comment:
Comparing floating number with `extent[0] === mathLog(originalExtent[0]` may cause precision problems. And can you explain why do you remove `this._fixMin` here?